cmd/guacrest/cmd/server.go

```diff
@@ -90,6 +96,37 @@ func startServer() {
 	cf()
 }
 
+// get the service handler
+// if an ent address is provided, get the handler backed by ent
+func getRestApiHandlerOrExit(ctx context.Context, gqlClient graphql.Client) gen.StrictServerInterface {
+	logger := logging.FromContext(ctx)
+	if flags.dbDirectConnection {
+		logger.Infof("Directly connecting to the Ent backend for optimized endpoint implementation")
+		ent := getEntClientOrExit(ctx)
+		handler := server.NewEntConnectedServer(ent, gqlClient)
+		return handler
+	}
+	return server.NewDefaultServer(gqlClient)
+}
+
+func getEntClientOrExit(ctx context.Context) *ent.Client {
+	logger := logging.FromContext(ctx)
+
+	if flags.dbDriver != dialect.Postgres {
+		logger.Warnf("Only Postgres is currently supported")
```

Comment:
nit: start with lowercase on messages, could also use `Fatalf` here, probably would be and error level log than a warn level log since it is a failure which prevents it from running.